applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_elimination_builder_and_solver.h

```diff
@@ -301,10 +303,12 @@ class TrilinosResidualBasedEliminationBuilderAndSolver
         for (typename ConditionsArrayType::ptr_iterator it=ConditionsArray.ptr_begin(); it!=ConditionsArray.ptr_end(); ++it)
         {
             //calculate elemental contribution
-            pScheme->Condition_Calculate_LHS_Contribution(*it,LHS_Contribution,EquationId,CurrentProcessInfo);
+            pScheme->Calculate_LHS_Contribution(**it,LHS_Contribution,EquationId,CurrentProcessInfo);
```

Comment:
Do we need underscores still ?

Reply:
In new code I would remove them, but changing this would imply to change a lot of (sensitive) code only for a minor naming issue

Reply:
Well .. the names are also changed in this PR. Anyway, when is the new code planned ?

Reply:
turned out that it was very little effort to change :D